### ISSUE TITLE
Add Robot Template for the Nutrition Schema Framework

### ISSUE THREAD

**Issue by kaiiam** [Southern-Cross-Plant-Science/cdno#9]
We'll be making use of a [Robot template](https://robot.obolibrary.org/template) to generate the Nutrition Schema Framework terms. As per FOODON's example, we'll use this [google sheet](https://docs.google.com/spreadsheets/d/1n6gRNaoV4SDl0mUzN_8HSOn--VMo6fbePKdhM4d1G10/edit?usp=sharing) to collaboratively edit the Robot template, and export it as a csv file and compile within the ontology.

**Comment by kaiiam** [comment]
This will be very similar to the work done in https://github.com/UA-SRC-data/srpdio/pull/7 and https://github.com/FoodOntology/foodon/pull/116.

**Comment by kaiiam** [comment]
Testing this in branch 9.

**Comment by kaiiam** [comment]
Trying it with the test output of a python script converting unreleased version of CDNO_nutritional_components_framework_v7.9 from @LilyAndres. 

```
robot template --template CDNO_nutritional_components_framework_v7.9_test.tsv -i ../cdno-edit.owl --prefix "CDNO:http://purl.obolibrary.org/obo/CDNO_" --prefix "CHEBI:http://purl.obolibrary.org/obo/CHEBI_" --prefix "ENVO:http://purl.obolibrary.org/obo/ENVO_"  -o CDNO_nutritional_components_framework_v7.9_test.owl
```

**Comment by kaiiam** [comment]
Doing it hopefully one last time with the full version of the output CDNO_nutritional_components_framework_v7.9

```
robot template --template CDNO_NCF_v7.9_output_for_ROBOT.tsv -i ../cdno-edit.owl --prefix "CDNO:http://purl.obolibrary.org/obo/CDNO_" --prefix "CHEBI:http://purl.obolibrary.org/obo/CHEBI_" --prefix "ENVO:http://purl.obolibrary.org/obo/ENVO_"  -o CDNO_NCF_v7.9_output_for_ROBOT.owl
```

**Comment by kaiiam** [comment]
Splitting it up into the two sheets one for the hierarchical NCF with is a relationships and one for the derivative terms. The rational is that following Chris Mungalls advice any terms with an equivalence axiom should follow a design pattern analogous to a DOSDP. Here it's a python script generating the formulaic input for a Robot template, but the result is very similar to using a DOSDP where it's programmatic based on a compositional combination of inputs. 

NCF terms:

```
robot template --template nutritional_components_framework.tsv -i ../cdno-edit.owl --prefix "CDNO:http://purl.obolibrary.org/obo/CDNO_" --prefix "CHEBI:http://purl.obolibrary.org/obo/CHEBI_" --prefix "ENVO:http://purl.obolibrary.org/obo/ENVO_"  -o nutritional_components_framework.owl
```


Sugar derivative terms:

```
robot template --template sugar_derivatives.tsv -i ../cdno-edit.owl --prefix "CDNO:http://purl.obolibrary.org/obo/CDNO_" --prefix "CHEBI:http://purl.obolibrary.org/obo/CHEBI_" --prefix "ENVO:http://purl.obolibrary.org/obo/ENVO_"  -o sugar_derivatives.owl
```

**Comment by kaiiam** [comment]
Addressed in #12 instead of here, can close this issue and #10.
